### ISSUE TITLE
Typo in vars for Magento_Catalog breakpoints in blank theme etc/view.xml 

### ISSUE THREAD

**Issue by koenner01** [magento/magento2#6820]
There is a typo in the theme-frontend-blank module in the etc/view.xml file.
In the options of the breakpoint block an option with name 'navigation' is used.
### Preconditions
1. Magento CE 2.1.1
### Steps to reproduce
1. Setup a new Magento 2 with CE 2.1.1
2. Set the frontend theme to the Blank theme
3. Go to a product detail page
4. Resize browser to width smaller than 767px
### Expected result

Media gallery thumbs should switch navigation to dots
### Actual result

Media gallery thumbs do not switch to dots

To fix this, the breakpoints block should use "nav" instead of "navigation"

```
        <var name="breakpoints">
            <var name="mobile">
                <var name="conditions">
                    <var name="max-width">767px</var>
                </var>
                <var name="options">
                    <var name="options">
                        <var name="nav">dots</var>
                    </var>
                </var>
            </var>
        </var>
```

**Comment by anton-huz** [comment]
Thanks for the catch!
We will deliver fix under MAGETWO-59527 internal issue.

**Comment by koenner01** [comment]
This seems to be fixed in develop https://github.com/magento/magento2/commit/104563f57c232b9f52861bbf0b9b8a4915273b35, but still present in MG2.1.4 CE

**Comment by magento-engcom-team** [comment]
@koenner01, thank you for your report.
The issue is already fixed in develop branch, 2.2.0
